src/canvas-extractor/tests/mapping/test_assignments.py

```diff
@@ -349,18 +350,33 @@ def assignment_dfs() -> Dict[str, DataFrame]:
                 "start_at": [SECTION_START_AT, SECOND_SECTION_START_AT],
                 "end_at": [SECTION_END_AT, SECOND_SECTION_END_AT],
                 "created_at": [SECTION_CREATED_AT, SECOND_SECTION_CREATED_AT],
-                "created_at_date": [SECTION_CREATED_AT_DATE, SECOND_SECTION_CREATED_AT_DATE],
+                "created_at_date": [
+                    SECTION_CREATED_AT_DATE,
+                    SECOND_SECTION_CREATED_AT_DATE,
+                ],
                 "restrict_enrollments_to_section_dates": [
                     SECTION_RESTRICT_ENROLLMENTS_TO_SECTION_DATES,
                     SECOND_SECTION_RESTRICT_ENROLLMENTS_TO_SECTION_DATES,
                 ],
-                "nonxlist_course_id": [SECTION_NONXLIST_COURSE_ID, SECOND_SECTION_NONXLIST_COURSE_ID],
-                "sis_section_id": [SECTION_SIS_SECTION_ID, SECOND_SECTION_SIS_SECTION_ID],
+                "nonxlist_course_id": [
```

Comment:
what does "nonxlist" mean?

Reply:
An excellent question.  Per the [Canvas API docs on Sections](https://canvas.instructure.com/doc/api/sections.html): "The unique identifier of the original course of a cross-listed section"